Formula/kubernetes-cli.rb

```diff
@@ -2,8 +2,8 @@ class KubernetesCli < Formula
   desc "Kubernetes command-line interface"
   homepage "https://kubernetes.io/"
   url "https://github.com/kubernetes/kubernetes.git",
```

Comment:
Looks like they have tagged releases now, so it'd probably be good to move to those: https://github.com/kubernetes/kubernetes/archive/v1.8.2.tar.gz

Reply:
@woodruffw Can't, as the version output of the program version depends on the git tree/hash. The test of this formula uses this too.

Reply:
See also f1238c1e38981baabd66cbbb2b445cd6e4b47d7d
Though the commit message could be a little bit more explicit :)

Reply:
Cool; thanks for the explanation @cblecker and @iMichka!